pos_session_closing_stock_error/models/pos_config.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    allow_session_closing_with_stock_errors = fields.Boolean(
```

Comment:
Hi. I think that the field should be set to default=False. Otherwise, the module has no effect, once installed. It is attended that it enables the feature, I think.
It is also more secure. What do you think ?

Reply:
> "It is attended that it enables the feature"

Yea.. sometimes I find myself in a dilemma because of this.
But I agree, most modules in this repo enable the feature upon install, so this one should do it too. :+1: